spec/controllers/email_preferences_controller_spec.rb

```diff
@@ -2,14 +2,27 @@
 
 describe Users::EmailPreferencesController do
   describe "edit" do
-    let(:user) { create (:user) }
+    let(:user) { create :user, angular_ui_enabled: true }
 
-    it 'renders the beta interface when an unsubscribe token is present' do
+    it 'renders the beta interface for betea user when an unsubscribe token is present' do
```

Comment:
I would love to have a 10 minute vebal run through what will be required. Not blocking at all - but it's something that could be done any time and really improve things